app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The use of a class-level dictionary `people` to store instances by name assumes that all names are unique. This can lead to incorrect spouse assignments if names are not unique. Consider using unique identifiers or implementing a strategy to handle name collisions.